tests/integration/targets/aci_vrf/tasks/main.yml

```diff
@@ -62,6 +63,7 @@
     policy_control_direction: egress
     preferred_group: enabled
     match_type: all
+    dataplane: "{{ fake_var | default(omit) }}"
```

Comment:
Why put it as fake_var?

Reply:
I don’t want to keep the dataplane while creating another vrf.